src/formatDate.js

```diff
@@ -8,7 +8,75 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
-}
+  // create property for symbols from and to
+  const simbolFormat = fromFormat[fromFormat.length - 1];
```

Comment:
The variable name 'simbolFormat' seems to be a typo. Consider renaming it to 'symbolFormat' for better readability.